base-server/index.js

```diff
@@ -617,6 +681,51 @@ export class BaseServer {
     }
   }
 
+  onSync(action, meta) {
+    if (this.actionsInQueue.has(meta.id)) {
+      return
+    }
+
+    let clientId = parseId(meta.id).clientId
+    let queueName = ''
+
+    let isChannel =
+      (action.type === 'logux/subscribe' ||
+        action.type === 'logux/unsubscribe') &&
+      action.channel
+
+    if (isChannel) {
+      for (let i = 0; i < this.channels.length && !queueName; i++) {
+        let channel = this.channels[i]
+        let pattern = channel.regexp || channel.pattern.regex
+        if (action.channel.match(pattern)) {
+          queueName = this.channelPatternToQueueName.get(pattern)
+        }
+      }
+    } else {
+      queueName = this.actionTypeToQueueName.get(action.type)
+    }
+
+    queueName = queueName || 'main'
+
+    let queueKey = `${clientId}/${queueName}`
+    let queue = this.queues.get(queueKey)
+
+    if (!queue) {
+      let concurrency = 1
```

Comment:
I am not sure that we need this variable, just call `queue = fastq(queueWorker, 1)`

Some developer will not understand what `1` means, but:
1. We don’t work with concurrency here
2. They have types for `fastq`